src/test/java/org/fogbowcloud/manager/SharedOrderHoldersTest.java

```diff
@@ -27,39 +21,42 @@
 
     @Before
     public void initialize() {
-        DatabaseManager databaseManager = Mockito.mock(DatabaseManager.class);
-        when(databaseManager.readActiveOrders(OrderState.OPEN)).thenReturn(new SynchronizedDoublyLinkedList());
-        when(databaseManager.readActiveOrders(OrderState.SPAWNING)).thenReturn(new SynchronizedDoublyLinkedList());
-        when(databaseManager.readActiveOrders(OrderState.FAILED)).thenReturn(new SynchronizedDoublyLinkedList());
-        when(databaseManager.readActiveOrders(OrderState.FULFILLED)).thenReturn(new SynchronizedDoublyLinkedList());
-        when(databaseManager.readActiveOrders(OrderState.PENDING)).thenReturn(new SynchronizedDoublyLinkedList());
-        when(databaseManager.readActiveOrders(OrderState.CLOSED)).thenReturn(new SynchronizedDoublyLinkedList());
-
-        PowerMockito.mockStatic(DatabaseManager.class);
-        given(DatabaseManager.getInstance()).willReturn(databaseManager);
+        mockReadOrdersFromDataBase();
 
         this.instanceOne = SharedOrderHolders.getInstance();
         this.instanceTwo = SharedOrderHolders.getInstance();
     }
 
+    // test case: As SynchronizedDoublyLinkedList is a sigleton object, when getting the
+    // list twice (or more) it must point to the same reference, in other words,
+    // they are the same object.
     @Test
     public void testGetSameListReference() {
+        // set up
         SynchronizedDoublyLinkedList listFromInstanceOne = instanceOne.getOpenOrdersList();
         SynchronizedDoublyLinkedList listFromInstanceTwo = instanceTwo.getOpenOrdersList();
-        assertEquals(listFromInstanceOne, listFromInstanceTwo);
 
+        // verify
+        Assert.assertEquals(listFromInstanceOne, listFromInstanceTwo);
```

Comment:
Should we test this using double equals signs (==) instead of equals? See @[this post](https://stackoverflow.com/questions/7520432/what-is-the-difference-between-vs-equals-in-java) for more info

Reply:
In Junit we have assertSame (same of ==) which assert that two objects refer to the same object. I will fix it.